Cargo.toml

```diff
@@ -22,3 +22,11 @@ inherits = "release"
 debug = true
 debug-assertions = true
 overflow-checks = true
+
+[patch.crates-io]
+winter_air = { git = "https://github.com/facebook/winterfell", package = "winter-air" }
+winter_math = { git = "https://github.com/facebook/winterfell", package = "winter-math" }
+winter_crypto = { git = "https://github.com/facebook/winterfell", package = "winter-crypto" }
+winter_prover = { git = "https://github.com/facebook/winterfell", package = "winter-prover" }
+winter_verifier = { git = "https://github.com/facebook/winterfell", package = "winter-verifier" }
+winter_utils = { git = "https://github.com/facebook/winterfell", package = "winter-utils" }
```

Comment:
The updated versions of winterfell crates are now on crates.io - so, this may not be necessary.

Reply:
removed